solution/ui/regulations/eregs-vite/src/views/PolicyRepository.vue

```diff
@@ -41,6 +46,28 @@ const props = defineProps({
     },
 });
 
+provide("apiUrl", props.apiUrl);
+provide("base", props.homeUrl);
```

Comment:
use [provide/inject](https://vuejs.org/guide/components/provide-inject.html) for non-reactive properties so that we don't have to prop drill them deep down to descendant components.